spec/integration/_dev/benchmark/spec.yml

```diff
@@ -6,11 +6,13 @@ spec:
       type: folder
       name: system
       required: false
-      additionalContents: false
+      additionalContents: true
```

Comment:
It looks like this is set to true because it needs to be added the `tcp-benchmark` folder, is that right ? 

Could there be more folders like this one under `system` or just one? 
Could it be added somehow to the specification? Maybe renaming it to a more generic name if it is just one folder expected for that.

Reply:
Probably, it could be used `pattern` to define the folders. Something like
https://github.com/elastic/package-spec/blob/a105f6aaec9460cfa5e601e7dac853871b36e6d4/spec/integration/data_stream/_dev/test/system/spec.yml#L6

Reply:
See my previous comment